website/client/components/inventory/equipment.vue

```diff
@@ -65,12 +65,15 @@
           v-for="(label, group) in gearTypesToStrings",
           :key="group",
           :item="flatGear[activeItems[group]]",
+          :showPopover="flatGear[activeItems[group]] && flatGear[activeItems[group]].key.indexOf('_base_0') === -1",
```

Comment:
Ah I get what you meant before, actually here `item` will already be avaible or at least it should be but I'm not super secure